quiz.js

```diff
@@ -0,0 +1,57 @@
+var button = document.getElementById("growTree");
+button.addEventListener("click", printTreeInfo); //event listenter
+
+function printTreeInfo() {
+  var buildingTree = { height: parseInt(document.getElementById("height").value),
+    symbol: document.getElementById("symbol").value
+  };
+  checkingUserInput(buildingTree);
+}
+
+function checkingUserInput(mytree) {
+
+  createTree(mytree);
```

Comment:
Looks like this is where the validation of your inputs should be, but it's missing.  Lets add the validation in.